miso-web/src/main/webapp/scripts/freezer_ajax.js

```diff
@@ -284,7 +335,12 @@
   function displayLooseStorage(storage) {
     var table = $('#levelTwoStorageLayout');
     var row = $('<tr>');
-    var cell = $('<td>').text('(unorganised space)')
+    var cell = $('<td>');
+    for(box in storage.boxes){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Add the "let", "const" or "var" keyword to this declaration of "box" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2703)